_develop/cross-compiling.md

```diff
@@ -31,5 +31,39 @@ If further errors occur it is likely to be due to missing packages. Some target
 
 ### Using a virtual environment
 
-As a Linux system is able to cross compile to macOS and Windows easily it can be simpler to use a virtualised environment when you are not developing from Linux. Docker images are a useful tool for a complex build configuration and this works for Fyne as well. Documentation for this process will follow...
+As a Linux system is able to cross compile to macOS and Windows easily it can be simpler to use a virtualised environment when you are not developing from Linux. Docker images are a useful tool for a complex build configuration and this works for Fyne as well. There are different tools that can be used. The tool recommended by the Fyne developers is [fyne-cross](https://github.com/lucor/fyne-cross). It has been inspired by [xgo](https://github.com/karalabe/xgo) and uses a [docker image](https://hub.docker.com/r/lucor/fyne-cross) built on top of the [golang-cross](https://github.com/docker/golang-cross) image,
+that includes the MinGW compiler for windows, and an OSX SDK, along the Fyne requirements.
```

Comment:
OSX should read "mac OS".
Note also that "along the Fyne requirements" is probably missing a word.

Reply:
Updated, thanks!